README.md

```diff
@@ -41,11 +41,51 @@ The parameters are as follows:
 
  As a module, this library also exports two helper functions:
 
+**`addOverrides`:**
```

Comment:
`addOverrides` works, but curious why we moved away from `mergeOverrides` as mentioned in https://github.com/stamen/mapbox-gl-style-build/issues/9?

To me `mergeOverrides` feels more accurate, but I don't have strong opinions here.